__tests__/e2e/species.spec.ts

```diff
@@ -1,47 +1,48 @@
+import exampleOrganizations from '@mocks/organizations/exampleOrganization.json';
+import examplePlanter from '@mocks/planters/examplePlanter.json';
+import exampleSpecies from '@mocks/species/exampleSpecies.json';
```

Comment:
@QuaidBartolomei what's this interesting thing about?

Reply:
I need to seed data into the db that is being created in the new pr workflow. The existing json objects are outdated and unable to be inserted so I created some new ones. The `@mocks/` path is a path alias I added to jest.config.js so I don't have to use the relative paths